src/routes/klageskjema/begrunnelse/begrunnelse.tsx

```diff
@@ -194,7 +196,7 @@ const Begrunnelse = ({ klage }: Props) => {
             <Reasons checkedReasons={reasons} setCheckedReasons={setReasons} />
 
             <Section>
-                <KlageUndertittel>Vedtaksdato (valgfri)</KlageUndertittel>
+                <FieldLabel>Vedtaksdato (valgfri)</FieldLabel>
```

Comment:
Siden vi bytter ut en `h2` med en "label", tenker jeg det er lurt å bruke en faktisk `label`.
Da blir den lik som `saksnummer` under også.

I tillegg må vi da knytte sammen `Datepicker` og `label` slik at de fungerer likt som `saksnummer`.

Reply:
`Datepicker` har en egenskap kalt `inputId` som vi kan bruke til å knytte det sammen med en `label` med `for`-egenskapen.

Reply:
> `Datepicker` har en egenskap kalt `inputId` som vi kan bruke til å knytte det sammen med en `label` med `for`-egenskapen.

Okay, skal kikke på det.